cmake/catkin_package.cmake

```diff
@@ -432,6 +432,31 @@ function(_catkin_package)
   # package cmake dir is the folder where the generated pkgConfig.cmake is located
   set(PKG_CMAKE_DIR "\${${PROJECT_NAME}_DIR}")
 
+  # strip SYSROOT prefixes for each element in LIST
+  macro(catkin_strip_sysroot)
+    cmake_parse_arguments(_fn "UNIQUE" "LIST" "" ${ARGN})
+    set(_list ${${_fn_LIST}})
+    set(${_fn_LIST})
+    foreach(_ele ${_list})
+      if(IS_ABSOLUTE ${_ele})
+        foreach(_path ${CMAKE_FIND_ROOT_PATH})
+          string(REGEX REPLACE "^${_path}" "" _ele ${_ele})
```

Comment:
If I understand this correctly, this should be fixed by doing something like
```cmake
string(REGEX REPLACE "^${_path}/" "/" _ele ${_ele})
```
(just off the top of my head, without testing)